selfdrive/car/hyundai/values.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, CP):
     self.STEER_STEP = 1  # 100 Hz
 
     if CP.carFingerprint in CANFD_CAR:
-      self.STEER_MAX = 270
+      self.STEER_MAX = 170
```

Comment:
Looked at this by request of @johnnathanwood. KONA_2022 isn't a CAN_FD car, so this change wasn't effective and he's still testing on full torque. It falls under ALT_LIMITS.

@sshane @sunnyhaibin can one of you push a test update to this branch lowering KONA_2022 to like 190?

Reply:
> Looked at this by request of @johnnathanwood. KONA_2022 isn't a CAN_FD car, so this change wasn't effective and he's still testing on full torque. It falls under ALT_LIMITS.
> 
> @sshane @sunnyhaibin can one of you push a test update to this branch lowering KONA_2022 to like 190?

@jyoung8607 is this to test the "oversteer" behavior?